src/fixes.rs

```diff
@@ -143,7 +145,7 @@ mod tests {
             ),
         ];
 
-        assert_eq!(0, run(&mut warnings, &mut lines, &[]));
+        assert_eq!(2, run(&mut warnings, &mut lines, &[]));
```

Comment:
Could you tell me more about this change?
Has this test been wrong before?

Reply:
Actually, this test seemed a bit strange. But yeah, previously, warnings were not fixed if the order of the lines was wrong.  

I think this behavior makes sense now.  Because, the warnings are attached to the original lines, this should not depend on `LineEntry` ordering

In fact, now UnorderedKeyFixer should not break the behavior of others.